### PR TITLE
Add mscheltienne to codeowners

### PR DESCRIPTION
**Pull request by mscheltienne**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -74,8 +74,11 @@
 /mne/minimum_norm @agramfort
 
 # Preprocessing
-/mne/preprocessing/ica.py @cbrnr @adam2392 @agramfort
-/mne/preprocessing/infomax_.py @cbrnr @adam2392
+/mne/preprocessing/ica.py @cbrnr @adam2392 @agramfort @mscheltienne
+/mne/preprocessing/infomax_.py @cbrnr @adam2392 @mscheltienne
+/mne/preprocessing/.*annotate.*py @mscheltienne
+/mne/preprocessing/bads.py @mscheltienne
+/mne/preprocessing/e.g.py @mscheltienne
```
Comment (hoechenberger) [inline .github/CODEOWNERS:81]: Is this ... is this a mistake?

Comment (larsoner) [reply .github/CODEOWNERS:81]: Unfortunately this is `gitignore` syntax not regex so you'll need:
```suggestion
/mne/preprocessing/e*g.py @mscheltienne
```
Checked by copying `CODEOWNERS` to `.gitignore`, removing `@mscheltienne` from this line, and running
```
$ git check-ignore --no-index --verbose mne/preprocessing/ecg.py
.gitignore:81:/mne/preprocessing/e*g.py	mne/preprocessing/ecg.py
```
it didn't return anything with the `.` in there

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -74,8 +74,11 @@
 /mne/minimum_norm @agramfort
 
 # Preprocessing
-/mne/preprocessing/ica.py @cbrnr @adam2392 @agramfort
-/mne/preprocessing/infomax_.py @cbrnr @adam2392
+/mne/preprocessing/ica.py @cbrnr @adam2392 @agramfort @mscheltienne
+/mne/preprocessing/infomax_.py @cbrnr @adam2392 @mscheltienne
+/mne/preprocessing/.*annotate.*py @mscheltienne
```
Comment (larsoner) [inline .github/CODEOWNERS:79]:
```suggestion
/mne/preprocessing/*annotate*.py @mscheltienne
```

**Comment by larsoner** [comment]
Thanks @mscheltienne !

**Pull request merged**